datasets/meg.py

```diff
@@ -1,34 +0,0 @@
-from benchopt import BaseDataset
```

Comment:
this should not be deleted

Reply:
My bad, I just checked on openML and saw classification tasks so I assumed this dataset was not a regression one

Reply:
can someone update openml to say it's regression?

Reply:
@agramfort I have the rights on this dataset

@TheoGuyard Can you show me where it shows as classif https://www.openml.org/search?type=data&status=active&id=43884 ?

Reply:
fixed